components/Cards/SchoolCard/__stories__/SchoolCard.stories.js

```diff
@@ -26,6 +26,7 @@ storiesOf('Cards/SchoolCard', module)
       logoSource={text('logoSource', `${s3}codeSchoolLogos/general_assembly.jpg`)}
       name={text('name', 'General Assembly')}
       website={text('website', 'https://generalassembly.com')}
+      toggleModal={() => {}}
```

Comment:
In the scope of each `add` method's callback, you can define a prop using knobs ahead of time and use it in two places.

So, you could create variables for the `name` and the `locations` before the JSX and use it in the `toggleModal` AND the original props. Value changes in the knobs would then reflect in the modal and the card.